spec/reciter_spec.rb

```diff
@@ -29,4 +25,18 @@
 
   end
 
+  describe '#recite_semirandom_rhyme' do
+
+    it 'returns the rhyme with each element in semi-random order' do
+      semirandom_indices1 = [0, 7, 10, 6, 4, 5, 2, 3, 11, 8, 1, 9]
```

Comment:
praise: This is cool! I hadn't thought of providing the indices as a way to make a stable test.